js/examples/next-openai-telemetry-app/pnpm-lock.yaml

```diff
@@ -0,0 +1,3853 @@
+lockfileVersion: '6.0'
```

Comment:
@Parker-Stafford intentional?

Reply:
yesi think this is the version we should be on since we are on pnpm 8 will check top level lock-file